### ISSUE TITLE
Error reading opencv module

### ISSUE THREAD

**Issue by denismakogon** [bytedeco/javacpp-presets#777]
Observing this for the last couple hours:
```
Error reading module: home/root/.m2/repository/org/bytedeco/opencv/4.1.1-1.5.2-SNAPSHOT/opencv-4.1.1-1.5.2-SNAPSHOT.jar
```
Tried on both fresh new docker Oracle Linux, MacOS. The same error.

**Comment by denismakogon** [comment]
So, it's gone, but for how long? Is it something related to the repository or what?

**Comment by saudet** [comment]
Sonatype OSS is configured to remove automatically snapshots that are older than 3 days. We'd have to set up a Nexus server somewhere else to have more control.

**Comment by saudet** [comment]
Maybe this could help: https://github.com/features/package-registry

In any case, would be good to know what we can do with that.

**Comment by denismakogon** [comment]
One more thing, I noticed that `linux-x86_64` packages for opencv are gone (`4.1.0-1.5.2-SNAPSHOT`), meaning:
```xml
        <dependency>
            <groupId>org.bytedeco</groupId>
            <artifactId>opencv</artifactId>
            <version>4.1.0-1.5.2-SNAPSHOT</version>
            <classifier>linux-x86_64</classifier>
        </dependency>
```
no longer works, is it something expected?

**Comment by denismakogon** [comment]
`{platform}-gpu` packages are there, but the regular one has gone when `4.1.1` was published.

**Comment by saudet** [comment]
Yes, those are no longer getting updated and the server cleaned up (almost) everything.

**Comment by denismakogon** [comment]
The reason why I ask is that I've observed `malloc` problems with `4.1.1` on Oracle Linux 7 (had no issues with `4.1.0`), but can't provide traces because I lost them, sorry.

**Comment by saudet** [comment]
Nothing seeing any changes that could cause such regression:
https://github.com/opencv/opencv/wiki/ChangeLog
The issue is probably there with 4.1.0, whatever it is.

**Comment by saudet** [comment]
One major change though appears to be optimization for AVX512. If you need to run this on a machine that supports AVX512, but might want to try and disable that.

**Comment by saudet** [comment]
In any case, the bug has probably already been fixed upstream.
Please try again with OpenCV 4.3.0: https://search.maven.org/search?q=org.bytedeco%20opencv
